src/entities/IBeacon.ts

```diff
@@ -19,6 +20,7 @@ export interface IBeacon {
   uses: IUse[];
   owners: IOwner[];
   emergencyContacts: IEmergencyContact[];
+  links: ILink[];
```

Comment:
`links` and `ILink` have a subtle naming collision with `<Link>` React elements of react-router-dom and material UI.  It may not be that much of an issue because domain object attribute and React component are quite separate concepts...  

What do you think? Might we name this property and its interface `linkedEntities`, `associations`, or something else?